convert2rhel/systeminfo.py

```diff
@@ -395,6 +422,28 @@ def _is_dbus_running(self):
 
         return status
 
+    def get_system_release_info(self, system_release_content=None):
+        """Return the system release information as an dictionary
+
+        This function aims to retrieve the system release information in an dictionary format.
+        This can be used before and after we modify the system-release file on the system,
+        as it have a parameter to to read from the contents of a system-release file (if called from somewhere else).
+
+        :param system_release_content: The contents of the system_release file if needed.
+        :type refresh_system_release_content: str
+        :returns: A dictionary containing the system release information
+        :rtype: dict[str, str]
+        """
+        distribution_id = self._get_system_distribution_id(system_release_content)
+        distribution_name = self._get_system_name(system_release_content)
+        distribution_version = self._get_system_version(system_release_content)
```

Comment:
A note about how we get the version here… Using the `self._get_system_version()` function, we will lose the patch number in a version, for example: 

```
7.6.1810 -> 7.6
```
The `.1810` part will be cut off, as it returns an instance of the class `Version` that currently only matches the first two part. If that's not a problem, I would say we can continue with this and change in the future, or, we could go ahead in this PR and adapt the `Version` class to accept one more version (`Version.patch`) and mark it as optional.